lib/internal/perf/performance_entry.js

```diff
@@ -91,10 +91,19 @@ function initPerformanceEntry(entry, name, type, start, duration) {
   entry[kDuration] = duration;
 }
 
+function FastPerformanceEntry(name, type, start, duration) {
+  initPerformanceEntry(this, name, type, start, duration);
+}
+
+ObjectSetPrototypeOf(FastPerformanceEntry.prototype, PerformanceEntry.prototype);
+ObjectSetPrototypeOf(FastPerformanceEntry, PerformanceEntry);
+
 function createPerformanceEntry(name, type, start, duration) {
-  return ReflectConstruct(function PerformanceEntry() {
-    initPerformanceEntry(this, name, type, start, duration);
-  }, [], PerformanceEntry);
+  const entry = new FastPerformanceEntry(name, type, start, duration);
+
+  entry.constructor = PerformanceEntry;
```

Comment:
Is the WPT tests fails without this? If so, please add a comment that it's for spec compliance and if not than maybe remove

Reply:
I updated the entire PR, I read your implementation on https://github.com/nodejs/node/commit/9fc57006c27564ed7f75eee090eca86786508f51#diff-a75b6b92eada0f88527acce4428a28471b39067fd0ddaab2fcf26cc022bded47R679-R682, it got faster and cleaner in this way, nice work!